frontend/src/components/MapView/LeftPanel/layersPanel/MenuItem/MenuSwitch/SwitchItems.tsx

```diff
@@ -21,12 +22,18 @@ interface SwitchItemsProps extends WithStyles<typeof styles> {
   extent?: Extent;
 }
 const SwitchItems = ({ layers, extent, classes }: SwitchItemsProps) => {
+  const { selectedLayers } = useLayers();
   return (
     <>
       {layers.map((layer: LayerType) => {
         const foundNotRenderedLayer = layer.group?.layers.find(layerItem => {
-          return layerItem.id === layer.id && !layerItem.main;
+          return (
+            layerItem.id === layer.id &&
+            !layerItem.main &&
+            !selectedLayers.some(sl => sl.id === layerItem.id)
```

Comment:
Allows the non-"main" layer to show on initial load if it's the selected layer (relevant for SPI 3 months when 3 months is selected)